src/App.js

```diff
@@ -1,8 +1,53 @@
 import './index.css';
-import axios from 'axios';
-import { useEffect,useState } from 'react';
+import { useEffect, useState } from 'react';
```

Comment:
Your App.js looks good! Your routing def looks top tier lol. 
You can try removing `useEffect` since it's not being used in App.js